dandi/cli/formatter.py

```diff
@@ -22,21 +22,57 @@ class JSONFormatter(Formatter):
     def __init__(self, indent=None, out=None):
         self.out = out or sys.stdout
         self.indent = indent
+        self.first = True
 
     @staticmethod
     def _serializer(o):
         if isinstance(o, datetime.datetime):
-            return o.__str__()
+            return str(o)
+        return o
+
+    def __enter__(self):
+        print("[", end="", file=self.out)
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        if not self.first:
+            print(file=self.out)
+        print("]", file=self.out)
+
+    def __call__(self, rec):
+        import json
+        from textwrap import indent
+
+        if self.first:
+            print(file=self.out)
+            self.first = False
+        else:
+            print(",", file=self.out)
```

Comment:
would you be so kind to add a proper unittest which would operate over some very simple datastructures but with at least 2 items so we could ensure that we are formatting multi-item output correctly?

Reply:
Just for `JSONFormatter` or also for `JSONLinesFormatter`?

Reply:
thanks for the tests, please add one for `JSONLinesFormatter` as well while at it -- would only help

Reply:
Done.